### PR TITLE
Updates for trifecta tech ownership and sponsoring

### PR DESCRIPTION
**Pull request by erikjee**

### CODE REVIEW

File: README.md
Comment (hdoordt) [inline README.md:70]:
```suggestion
In addition, one of our maintainers, [@henk](https://github.com/hdoordt), received a grant from the [Rust Foundation](https://foundation.rust-lang.org/).
```

**Comment by erikjee** [comment]
Note: blocked by approval for github sponsoring / stripe

**Comment by erikjee** [comment]
@hdoordt FYI, I'm still waiting for the GitHub sponsoring / stripe approval. For some reason, it is taking unusually long. I'll comment here again when it is ready.

**Comment by erikjee** [comment]
@hdoordt GitHub sponsoring / stripe approval finally came through. Can you merge this now?

**Review by hdoordt — APPROVED** [review]

**Pull request merged**
